lib/pcb/svg-object-fns/create-svg-objects-from-pcb-courtyard-rect.ts

```diff
@@ -0,0 +1,93 @@
+import type { PcbCourtyardRect } from "circuit-json"
+import type { INode as SvgObject } from "svgson"
+import { applyToPoint } from "transformation-matrix"
+import type { PcbContext } from "../convert-circuit-json-to-pcb-svg"
+
+export function createSvgObjectsFromPcbCourtyardRect(
+  pcbCourtyardRect: PcbCourtyardRect,
+  ctx: PcbContext,
+): SvgObject[] {
+  const { transform, layer: layerFilter, colorMap } = ctx
+  const {
+    center,
+    width,
+    height,
+    layer = "top",
+    pcb_courtyard_rect_id,
+    stroke_width,
+    is_filled,
+    has_stroke,
+    is_stroke_dashed,
```

Comment:
can you update to the latest circuit-json, some of these properties were removed